tests/procurement/tenders/exemptions/fixtures/publisher_urls

```diff
@@ -0,0 +1,48 @@
+pipelines/procurement/tenders/exemptions/add_publisher_urls_resource
+--
+{"mock": true}
```

Comment:
Take a look at the work done here, perhaps it would be useful for mocking:
https://github.com/frictionlessdata/datapackage-pipelines/commit/748c067e87366b21404c39ff319c1240a964393f